lib/src/clipboard/clipboard.dart

```diff
@@ -0,0 +1,16 @@
+
+import 'package:flutter/services.dart';
+import 'package:flushbar/flushbar.dart';
+
+final flushbarCopied = Flushbar<dynamic>(
```

Comment:
I'm would have instanciated the Flushbar Object in the function but okay for this one